src/components/sidebar/core/DatabaseDropdown.vue

```diff
@@ -30,7 +40,11 @@
     methods: {
       async refreshDatabases() {
         this.dbs = await this.connection.listDatabases()
-      }
+      },
+      async databaseCreated() {
+        this.$modal.hide('config-add-database')
+        await this.refreshDatabases()
```

Comment:
I'd also suggest switching to the new database after creation. `this.currentDatabase = newDatabase` and pass that through into the created callback

Reply:
done